docs/content/release-notes/flink-1.18.md

```diff
@@ -0,0 +1,152 @@
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 1.18
+
+These release notes discuss important aspects, such as configuration, behavior or dependencies,
+that changed between Flink 1.17 and Flink 1.18. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.18.
+
+
+### Build System
+
+#### Support Java 17 (LTS)
+
+##### [FLINK-15736](https://issues.apache.org/jira/browse/FLINK-15736)
+Apache Flink was made ready to compile and run with Java 17 (LTS). This feature is still in beta mode. 
+Issues should be reported in Flink's bug tracker.
+
+
+### Table API & SQL
+
+#### Unified the max display column width for SqlClient and Table APi in both Streaming and Batch execMode
+
+##### [FLINK-30025](https://issues.apache.org/jira/browse/FLINK-30025)
+Introduction of the new ConfigOption DISPLAY_MAX_COLUMN_WIDTH (table.display.max-column-width) 
+in the TableConfigOptions class is now in place. 
+This option is utilized when displaying table results through the Table API and sqlClient. 
+As sqlClient relies on the Table API underneath, and both sqlClient and the Table API serve distinct 
+and isolated scenarios, it is a rational choice to maintain a centralized configuration. 
+This approach also simplifies matters for users, as they only need to manage one configOption for display control.
+
+During the migration phase, while sql-client.display.max-column-width is deprecated, 
+any changes made to sql-client.display.max-column-width will be automatically transferred to table.display.max-column-width. 
+Caution is advised when using the CLI, as it is not recommended to switch back and forth between these two options.
+
+#### Introduce Flink Jdbc Driver For Sql Gateway
+##### [FLINK-31496](https://issues.apache.org/jira/browse/FLINK-31496)
+Apache Flink now supports JDBC driver to access sql-gateway, you can use the driver in any cases that
+support standard JDBC extension to connect to Flink cluster.
+
+#### Extend watermark-related features for SQL
+##### [FLINK-31535](https://issues.apache.org/jira/browse/FLINK-31535)
+Flink now enables user config watermark emit strategy/watermark alignment/watermark idle-timeout
+in Flink sql job with dynamic table options and 'Options' hint.
```

Comment:
```suggestion
in Flink sql job with dynamic table options and `OPTIONS` hint.
```
Since most of the times in docs hints are mentioned in uppercase + added code mark